app/views/users/_admin_change_username.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2 class="heading"><%= t(".page_heading") %></h2>
+<%= error_messages_for :user %>
+<p class="note"><%= t(".description") %></p>
+
+<ul class="navigation actions">
+  <li><%= link_to t(".navigation.manage_user"), admin_user_path(@user) %></li>
+</ul>
+
+<%= form_tag changed_username_user_path(@user) do %>
+  <dl>
+    <dt><%= t("users.change_username.current_username") %></dt>
+    <dd><p class="important informational"><%= @user.login %></p></dd>
+    <dt><%= label_tag :new_login, t("users.change_username.new_username") %></dt>
+    <dd>
+      <%= text_field_tag :new_login, "user#{@user.id}", readonly: true, autocomplete: "off", "aria-describedby": "new-login-field-description" %>
+      <p class="footnote" id="new-login-field-description">
+        <%= t(".new_username_footnote") %>
+      </p>
+    </dd>
+    <dt><%= label_tag :ticket_number, t(".ticket_id") %></dt>
+    <dd><%= text_field_tag :ticket_number, nil, autocomplete: "off" %></dd>
+    <dt class="landmark"><%= t("users.change_username.submit_landmark") %></dt>
+    <dd class="submit actions">
+      <%= submit_tag t("users.change_username.change_username") %>
```

Comment:
`users.change_username.submit`? Makes it clearer where on the page it's used (i.e., it's the text of the submit button)